primefaces/src/main/java/org/primefaces/util/FileUploadUtils.java

```diff
@@ -86,29 +92,29 @@ private FileUploadUtils() {
      */
     public static String requireValidFilename(String filename) {
         if (LangUtils.isBlank(filename)) {
-            throw validationError("Filename cannot be empty or null");
+            throw validationError(MessageFactory.getMessage(FILE_LIMIT_KEY));
```

Comment:
why FILE_LIMIT?
sounds more like a FILENAME_EMPTY?

Reply:
Yep fixing it now